src/HornedBeasts.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+class HornedBeast extends React.Component{
+    render(){
+        return(
+            <article>
+                <h2>Rhino</h2>
+                <img
+                src="https://images.unsplash.com/photo-1605477899141-ac050a727db1?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTB8fHJoaW5vfGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=800&q=60"
+                alt="Rhino"
+                title="Rhino"
```

Comment:
- This is looking great! We can indent lines 9-11 👍  to keep in line with best practices